ci-operator/step-registry/cucushift/upgrade/helper-service/build-info/cucushift-upgrade-helper-service-build-info-commands.sh

```diff
@@ -9,6 +9,12 @@ BUILD_INFO_FILE="${ARTIFACT_DIR}/ocp-build-info"
 if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
     echo "Release info for: ${RELEASE_IMAGE_LATEST}"
     oc adm release info "${RELEASE_IMAGE_LATEST}" || true
-    LATEST_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
-    echo "${LATEST_VERSION}" > "${BUILD_INFO_FILE}"
+    INITIAL_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
+    echo "${INITIAL_VERSION}" >> "${BUILD_INFO_FILE}"
 fi
+if [[ -n "${RELEASE_IMAGE_TARGET}" ]]; then 
+    echo "Release info for: ${RELEASE_IMAGE_TARGET}"
+    oc adm release info "${RELEASE_IMAGE_TARGET}" || true
+    UPGRADED_VERSION="$(oc adm release info "${RELEASE_IMAGE_TARGET}" --output=jsonpath="{.metadata.version}")"
```

Comment:
Prefer to use var name `TARGET_VERSION` here.
Consistent naming can help reading and debugging.

Reply:
@liangxia Thanks, fixed it.